src/homeworkSecond/CheckingExpression.java

```diff
@@ -0,0 +1,14 @@
+package homeworkSecond;
+
+public class CheckingExpression {
+
+    public static void main(String[] args) {
+        System.out.println(doubleExpression(3.02, 5.03, 8.05));
+    }
+
+    private static final double error = 0.0001;
```

Comment:
constant names write with uppercase
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

Reply:
Yes of course! Excuse me, please!